src/BizHawk.BizInvoke/MemoryBlockBase.cs

```diff
@@ -142,7 +151,8 @@ public override long Position
 
 			private void EnsureNotDisposed()
 			{
-				if (_owner.AddressRange.Start == 0) throw new ObjectDisposedException(nameof(MemoryBlockBase)); //TODO bug?
```

Comment:
Regarding the TODO:  Yes, this does indicate a bug, but not here; in the calling code.  So nothing to change here.

Reply:
I was unclear: before I changed it, it accessed `_owner.Start`, which I assumed would return `default(int)` if `_owner` was GC'd. Now it's accessing `_owner.AddressRange`, which I assumed would return `default(Range<int>) == null` if `_owner` was GC'd. In both cases an exception is thrown by the method.

Reply:
What in the seven hells are you talking about?  `_owner` cannot possibly be GCed.  That's how C# works.  References don't disappear randomly; if it's observable by any code, it is not GCed.

Reply:
Not GC'd sorry, *disposed*. TBH I don't fully understand either mechanism.